shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -51,26 +61,36 @@ public String getCommandName() {
   }
 
   @Override
+  public Options getOptions() {
+    return new Options()
+        .addOption(LOCAL_OPTION);
+  }
+
+  @Override
   protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
-    load(path);
+    load(path, cl.hasOption(LOCAL_OPTION.getLongOpt()));
   }
 
   /**
-   * Loads a file or directory in Alluxio space, makes it resident in memory.
+   * Loads a file or directory in Alluxio space, makes it resident in Alluxio.
    *
-   * @param filePath The {@link AlluxioURI} path to load into Alluxio memory
+   * @param filePath The {@link AlluxioURI} path to load into Alluxio
+   * @param local the local flag; If the file is already in Alluxio fully, enable the local flag,
+   *              Alluxio will still load the file, otherwise Alluxio will do nothing.
```

Comment:
`@param local whether to load data to a local worker even when the data is already loaded remotely`